src/schema/v2/__tests__/marketingCollections.test.ts

```diff
@@ -172,4 +172,48 @@ describe("MarketingCollections", () => {
       curatedMarketingCollections: payload,
     })
   })
+
+  it("uses the custom sort for marketing collections", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CUSTOM, size: 2, category: "Gallery") {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
+
+    const context = {
+      authenticatedLoaders: {},
+      marketingCollectionsLoader: () => Promise.resolve({ body: payload }),
+    } as any
+
+    const data = await runQuery(query, context)
+
+    expect(data).toEqual({
+      marketingCollections: payload,
+    })
```

Comment:
**suggestion:** This test isn't really exercising anything other than that the mock data is being passed through. 

I think it would be better to test the actual side effects of passing the custom sort, e.g. assert that the final args to Gravity include a non-empty list of `slugs`, and do not include `category` or `sort`.

Reply:
Sounds good!